astro/src/content/docs/_shared/_common-configuration.mdx

```diff
@@ -19,6 +19,18 @@ Create a second user with the admin role. This user can provide access should yo
 * Add a user with a known, valid email address and secure password within your organization.
```

Comment:
On line 1 of this file, please add

import Breadcrumb from 'src/components/Breadcrumb.astro';

And make sure this renders correctly.

Reply:
Added in there, looks normal on my end.